Expecta.xcodeproj/project.pbxproj

```diff
@@ -271,6 +271,8 @@
 		908379DA1A8B984F009844DA /* NSValue+Expecta.m in Sources */ = {isa = PBXBuildFile; fileRef = E9ACDF5713B2DDC60010F4D7 /* NSValue+Expecta.m */; };
 		908379DE1A8B9978009844DA /* Expecta.h in Headers */ = {isa = PBXBuildFile; fileRef = 908379DD1A8B990A009844DA /* Expecta.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		908379DF1A8B997D009844DA /* Expecta.h in Headers */ = {isa = PBXBuildFile; fileRef = 908379DD1A8B990A009844DA /* Expecta.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		94B11BF72590A9E300188B44 /* EXPDifference.h in Headers */ = {isa = PBXBuildFile; fileRef = 94B11BF62590A9E300188B44 /* EXPDifference.h */; };
+		94B11BFF2590A9FA00188B44 /* EXPDifference.h in Headers */ = {isa = PBXBuildFile; fileRef = 94B11BF62590A9E300188B44 /* EXPDifference.h */; };
```

Comment:
what are the different places this file is added to?

Reply:
It was added to Expecta and Expecta-iOS. Removed from Expecta.